README.md

```diff
@@ -322,6 +322,62 @@ Class['apt::update'] ->
 Class['::mysql::client']
 ```
 
+### Install MySQL 5.6 server on CentOS
+
+This minimal example shows how to install MySQL Community Server 5.6 on Centos 7.3 using Puppet 3.8.7 using hiera with puppetlabs-mysql=3.9.0
```

Comment:
My idea was to provide real working example of the code, so I've added those versions on purpose, because I did not have time to test it in different setup.
Also yum repo used in hiera explicitly defines MySQL version to use.

I hope this would be a good starting point for anyone else trying to define more generic MySQL section in the future.